package.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "web-ledger-context",
+  "version": "0.0.1-dev",
+  "description": "Web Ledger Context",
+  "main": "js",
```

`npm pack` is pulling in all the website stuff.  Only need to ship the `js` and `contexts` contents:
```suggestion
  "main": "js",
  "files": [
    "contexts",
    "js"
  ],
```